firebase_auth_oauth_web/pubspec.yaml

```diff
@@ -5,7 +5,7 @@ homepage: https://github.com/amrfarid140/firebase_auth_oauth/tree/master/firebas
 
 environment:
   sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=1.10.0"
+  flutter: ">=1.17.0"
```

Comment:
Why do we need this?

Reply:
If we changing it for the web plugin, would it be worth changing in the other 2 plugins as well?

Reply:
Oh, we don't need it, I made that locally and forgot to remove it. Pushed a version without it.